src/bokeh/plotting/_figure.py

```diff
@@ -369,7 +369,7 @@ def harea_stack(self, stackers, **kw):
         Examples:
 
             Assuming a ``ColumnDataSource`` named ``source`` with columns
-            *2016* and *2017*, then the following call to ``harea_stack`` will
```

Comment:
The consecutive "will" is fixed, but this fix is unrelated to the original issue.